RoddyCore/src/de/dkfz/roddy/config/ConfigurationValue.groovy

```diff
@@ -466,12 +416,12 @@ class ConfigurationValue implements RecursiveOverridableMapContainer.Identifiabl
 
     @Deprecated
     boolean isInvalid() {
-        return invalid
+        return !valid
     }
 
     @Deprecated
     void setInvalid(boolean invalid) {
-//        this.invalid = invalid
+        // Not used anymore, invalid is not overriden.
```

Comment:
Hm. "isInvalid()" is still available (returns !valid) but setInvalid suddenly is a no-op. To me this sounds dangerous. Maybe better

```groovy
void setInvalid(boolean invalid) {
   valid = !invalid
}
```

Reply:
Removed the method and the field